### PR TITLE
add type_urls to extension schema

### PR DESCRIPTION
**Pull request by kyessenov**
Signed-off-by: Kuat Yessenov <kuat@google.com>

### CODE REVIEW

File: envoy.code.check/envoy/code/check/typing.py [hunk 1]
```diff
@@ -17,6 +17,7 @@ class BaseExtensionMetadataDict(TypedDict):
 
 class ExtensionMetadataDict(BaseExtensionMetadataDict, total=False):
     undocumented: bool
+    type_urls: Optional[List[str]]
```
Comment (phlax) [inline envoy.code.check/envoy/code/check/typing.py:20]: no need for `Optional` as the `TypedDict` is set to `total=False`

Comment (kyessenov) [reply envoy.code.check/envoy/code/check/typing.py]: Thanks. This is all new to me in python.

Comment (phlax) [reply envoy.code.check/envoy/code/check/typing.py]: yep, its fairly new to python

**Review by phlax — APPROVED** [review]
lgtm, thanks @kyessenov !

**Comment by phlax** [comment]
@kyessenov i will publish a new version of `envoy.code.check` as soon as this lands

this will trigger a dependabot update in the next ~24 hours - if you want i can raise a PR to include the change and we can land sooner

**Comment by kyessenov** [comment]
@phlax Thanks, no need to rush. I don't expect my Envoy PR to be merged this fast.

**Pull request merged**